test/kw/kw_persistence.h

```diff
@@ -0,0 +1,53 @@
+/**
+ * \file kw_server_data.h
+ *
+ * \copyright Copyright (C) 2019 Kano Computing Ltd.
+ *            License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+ *
+ * \brief Tests for the Mercury::KW - Kano World - functions
+ *
+ */
+
+#ifndef TEST_KW_PERSISTENCE
+#define TEST_KW_PERSISTENCE
+
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <mercury/kw/kw.h>
+
+#include <sys/stat.h>
+
+using testing::Eq;
+
+
+TEST(kw, login_data_filename_created)
+{
+    KanoWorld kw;
+    struct stat st;
+
+    EXPECT_GT (kw.data_filename.length(), 0);
+
+    // Make sure the cache file is not here
+    if (stat(kw.data_filename.c_str(), &st) != -1) {
+        unlink(kw.data_filename.c_str());
```

Comment:
Maybe erasing the store should be part of the API, i.e. create a `logout()`-type function which essentially removes this.

Reply:
Done - implemented a logout function